apps/site/src/_pages/docs/3_using-blocks.mdx

```diff
@@ -27,6 +27,10 @@ The [Block Protocol for Wordpress](https://blockprotocol.org/wordpress) plugin a
 
 ### Other environments
 
-Support for additional embedding environments will arrive throughout 2023.
+Support for additional embedding environments will arrive throughout 2023. If there are others you'd like to see us exploring, please [reach out](https://blockprotocol.org/contact) to us.
 
-If you're looking to implement support for the Block Protocol in another application or framework that you'd like to see mentioned here, read more about [embedding blocks](https://blockprotocol.org/docs/embedding-blocks) and feel free to [reach out](https://blockprotocol.org/contact) to us.
+### Your own application
+
+If you're interested in supporting the Block Protocol within your application now, allowing your users to insert and use blocks from the [Þ Hub](https://blockprotocol.org/hub), please [contact us](https://blockprotocol.org/contact) with more information about your intended use, and we'll be in touch.
```

Comment:
I think _in addition_ we should say something to the effect of "In due course, we'll be releasing public documentation covering this process, but while the protocol remains under heavy active development, please reach out directly."

I worry that in its current form it makes the protocol look like it's not really open, in spite of the fact it obviously is.

Reply:
Done in https://github.com/blockprotocol/blockprotocol/commit/a99d4f2457ac5c73af8b6b4d164455591eaf0b0a